src/makeCalculator.js

```diff
@@ -4,7 +4,25 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (num) => (calculator.result += num),
+    subtract: (num) => (calculator.result -= num),
+    multiply: (num) => (calculator.result *= num),
+    divide: (num) => (calculator.result /= num),
+    reset: () => {
+      calculator.result = 0;
+
+      return calculator;
+    },
```

Comment:
[CODE KNOWLEDGE] - Similarly, for the `reset` method, use the method shorthand syntax instead of an arrow function. This will keep the code consistent and allow the use of the 'this' keyword if needed in the future.